google/resumable_media/_helpers.py

```diff
@@ -95,7 +95,8 @@ def require_status_code(response, status_codes, get_status_code, callback=do_not
     """
     status_code = get_status_code(response)
     if status_code not in status_codes:
-        callback()
+        if status_code not in common.RETRYABLE:
+            callback()
```

Comment:
We need an additional test (or tests) below for the case being fixed here:  it should have a line like:

```python
        callback.assert_not_called()
```

Reply:
Added test cases for the retryable status codes. PTAL, thanks!